packages/create-svelte/shared/+playwright+default/tests/test.js

```diff
@@ -0,0 +1,6 @@
+import { expect, test } from '@playwright/test';
+
+test('about page has expected h1', async ({ page }) => {
```

Comment:
Does playwrigt support writing your code in TypeScript? If yes, should we add a variant for TS users?

Reply:
Yeah, playwright does [support typescript](https://playwright.dev/docs/test-typescript) ootb, down to the playwright config too.